couchdbkit/client.py

```diff
@@ -33,8 +33,8 @@
 from copy import deepcopy
 from itertools import groupby
 import json
-from mimetypes import guess_type
 import time
+from urlparse import urlparse
```

Comment:
Could you replace this with the Py3 compatible import?
https://docs.python.org/3/library/urllib.parse.html